sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/ConfigProperties.java

```diff
@@ -129,4 +181,40 @@ private static ConfigurationException newInvalidPropertyException(
         .filter(s -> !s.isEmpty())
         .collect(Collectors.toList());
   }
+
+  /** Returns the TimeUnit associated with a unit string. Defaults to milliseconds. */
+  private static TimeUnit getDurationUnit(String unitString) {
+    // If statement in likely-to-be-used order.
```

Comment:
Let's just go in order of size to reduce confusion when reading this code, the optimization of one less branch for going out-of-order is just not worth it.

Reply:
Removed